paasta_tools/cli/cmds/status.py

```diff
@@ -255,19 +237,20 @@ def paasta_status_on_api_endpoint(
     output: List[str],
     system_paasta_config: SystemPaastaConfig,
     verbose: int,
-    new: bool = False,
 ) -> int:
     output.append("    instance: %s" % PaastaColors.cyan(instance))
     client = get_paasta_oapi_client(cluster, system_paasta_config)
     if not client:
         print("Cannot get a paasta-api client")
         exit(1)
+
     try:
         status = client.service.status_instance(
             service=service,
             instance=instance,
             verbose=verbose,
-            new=new,
+            # remove once deprecated on server side
+            new=(system_paasta_config.get_paasta_status_version() == "new"),
```

Comment:
Will you follow up with another PR to delete the `new` parameter and the supporting code for new=False from `status_instance`?